### ISSUE TITLE
PageObject getElement throws NPE for invalid fieldName

### ISSUE THREAD

**Issue by bharathc02** [RationaleEmotions/SimpleSe#2]
Consider the JSON file contains element name as `bar`.
```
{
  "name": "FooPage",
  "defaultLocale": "en_US",
  "elements": [
    {
      "name": "bar",
      "locale": [
        {
          "name": "en_US",
          "locator": "//h1"
        }
      ],
      "wait": {
        "until": "visible",
        "for": 45
      }
    }
  ]
}
```
From the test class, perform a getElement operation on `PageObject` class by passing `fieldName` as `wrongbar`. Upon execution, the method [newRawElement](https://github.com/RationaleEmotions/SimpleSe/blob/375f13f2d328555cc4d7e76fd2b628228654b1b2/src/main/java/com/rationaleemotions/page/PageObject.java#L259) throws NullPointerException. 

Ideally, it should throw an exception as element name not found in JSON.

**Comment by krmahadevan** [comment]
Fixed and released as part of `1.0.2`. Please upgrade and retry.

**Comment by bharathc02** [comment]
@krmahadevan Verified
